lib/cable_ready/identifiable.rb

```diff
@@ -21,10 +23,26 @@ def dom_id(record, prefix = nil)
     end
 
     def identifiable?(obj)
-      obj.respond_to?(:to_dom_selector) ||
+      (
+        obj.respond_to?(:to_dom_selector) ||
         obj.respond_to?(:to_dom_id) ||
-        obj.is_a?(ActiveRecord::Relation) ||
-        obj.is_a?(ActiveRecord::Base)
+        ar_relation?(obj) ||
```

Comment:
the tests fails here because how inheritance in ruby will prevent methods on the CableReady to be accesible when method is included in a nested includes.

As in https://github.com/stimulusreflex/cable_ready/pull/222/files#diff-bb8e7da8f683e6a053f852e827078c740fd88706ae7ca90dfbb515ef417b09ecR7

this module is nested inside, when calling ar_reliation? it is not there so you get the error.... I think 🤔